tests/kernel/mem_protect/userspace/prj.conf

```diff
@@ -1 +1,4 @@
 CONFIG_ZTEST=y
+CONFIG_INIT_STACKS=y
+CONFIG_APPLICATION_DEFINED_SYSCALL=y
+CONFIG_THREAD_USERSPACE_LOCAL_DATA=y
```

Comment:
@andrewboie correct me if I am wrong:

```
config ERRNO
	bool "Enable errno support"
	default y
	select THREAD_USERSPACE_LOCAL_DATA if USERSPACE
```

So THREAD_USERSPACE_LOCAL_DATA is selected by default if ERRNO is selected, right? And that one is default=y

Reply:
what are you trying to say here? that I should not explicitly put dependencies in prj.conf and rely on defaults elsewhere that might change?

Reply:
Yeap, that's what I am trying to say. Obviously, I am wrong :)